docs/section6/Rule6-14.md

```diff
@@ -16,26 +16,41 @@
 **Evaluation Context:** Each Data Element  
 **Data Lookup:** None  
 
+**Function Call:**
+
+1. compare_schedules()
+2. match_data_element()
+
 ## Rule Logic:  
 
-- **Applicability Check 1:** ```if sum(space.floor_area for zone.space in P_RMR) < 5000:```  
+- **Applicability Check 1:** `if sum(space.floor_area for zone.space in P_RMR) < 5000:`
+
```

Comment:
I think here we need to make it clear that we are looking for the total area of ALL buidling segments.

Reply:
Updated to `for P_RMR...spaces`.